codersdk/deploymentconfig.go

```diff
@@ -165,6 +167,11 @@ type LoggingConfig struct {
 	Stackdriver *DeploymentConfigField[string] `json:"stackdriver" typescript:",notnull"`
 }
 
+type DangerousConfig struct {
+	AllowPathAppSharing         *DeploymentConfigField[bool] `json:"allow_path_app_sharing" typescript:",notnull"`
+	AllowPathAppSiteOwnerAccess *DeploymentConfigField[bool] `json:"allow_path_app_site_owner_access" typescript:",notnull"`
+}
```

Comment:
I think it would be nice to include the dangerous ratelimits flag in this config.